src/main/java/io/logz/guice/jersey/configuration/JerseyConfigurationBuilder.java

```diff
@@ -38,6 +39,11 @@ public JerseyConfigurationBuilder addPort(int port) {
         return this;
     }
 
+    public JerseyConfigurationBuilder addPortWithHttpConfiguration(int port, HttpConfiguration httpConfiguration) {
```

Comment:
It's too many flavors. Let's create a `ServerConnectorConfigurationBuilder` and do this right